contracts/programs/gumball-machine/src/lib.rs

```diff
@@ -10,18 +10,18 @@ use bubblegum::program::Bubblegum;
 
 use bubblegum::state::metaplex_adapter::MetadataArgs;
 use bytemuck::cast_slice_mut;
-use gummyroll::{
-    program::Gummyroll, state::CandyWrapper,
-};
+use gummyroll::{program::Gummyroll, state::CandyWrapper};
 use spl_token::native_mint;
 pub mod state;
 pub mod utils;
 
-use crate::state::{GumballMachineHeader, EncodeMethod, ZeroCopy};
+use crate::state::{EncodeMethod, GumballMachineHeader, ZeroCopy};
 use crate::utils::get_metadata_args;
 
 declare_id!("GBALLoMcmimUutWvtNdFFGH5oguS7ghUUV6toQPppuTW");
 
+const COMPUTE_BUDGET_ADDRESS: &str = "ComputeBudget111111111111111111111111111111";
```

Comment:
What is going on here???

Reply:
Need a way to check that the first instruction is compute budget program